factories/workflow_models.py

```diff
@@ -76,14 +76,3 @@ class Meta:
         model = WorkflowLevel2M
 
     name = 'Help Syrians'
-
-    @post_generation
```

Comment:
Delete the import if you don't use it anymore

Reply:
Done!